README.md

```diff
@@ -693,8 +693,8 @@ Enter value for this parameter if neighbors touching cells needs to be calculate
 __Features:__
 Comma separated list of features to be extracted. If all the features are required, then choose option __*all*__.
 
-__Csvfile:__
-There are 2 options available under this category. __*Separatecsv*__ - to save all the features extracted for each image in separate csv file. __*Singlecsv*__ - to save all the features extracted from all the images in the same csv file.
+__Outputtype:__
+There are 4 options available under this category. __*Separatecsv*__ - to save all the features extracted for each image in separate csv file. __*Singlecsv*__ - to save all the features extracted from all the images in the same csv file. __*Arrow*__ - to save all the features extracted from all the images in arrow format. __*Parquet*__ - to save all the features extracted from all the images in parquet format
```

Comment:
Could it be a bit better (clear): "__*Arrow*__ - to save all the features extracted from all the images in Apache Arrow format." ?

Reply:
Also, "__*Parquet*__ - to save all the features extracted from all the images in Apache Parquet format" ? Just for the part of the audience who lack familiarity with non-binary formats including Apache ones

Reply:
updated as requested